rdbms/src/main/java/org/spine3/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -101,25 +116,61 @@ public EventStorage createEventStorage() {
                 .newInstance(dataSource, entityStorage, false, getProjectionStorageQueryFactory(dataSource, projectionClass));
     }
 
+    /**
+     * Returns {@link AggregateStorageQueryFactory}.
```

Comment:
This is kind of cap-style doc. We should try to say something useful, f.e. `Creates a new factory which produces database queries`.